config/filter.d/domino-pop3.conf

```diff
@@ -0,0 +1,38 @@
+# Fail2Ban configuration file for IBM Domino POP3 task authentication failures
+#
+# Author: Christian Brandlehner
+#
+# $Revision: 001 $
+#
+# Configuration:
+# Set the following Domino Server parameters in notes.ini:
+#       console_log_enabled=1
+#       log_sessions=2
+#       POP3Debug=2
+# You also have to use a date and time format supported by fail2ban. Recommended notes.ini configuration is:
+#       DateOrder=DMY
+#       DateSeparator=-
+#       ClockType=24_Hour
+#       TimeSeparator=:
+
+[Definition]
+
+# Option:  failregex
+# Notes.:  regex to match the password failure messages in the logfile. The
+#          host must be matched by a group named "host". The tag "<HOST>" can
+#          be used for standard IP/hostname matching and is only an alias for
+#          (?:::f{4,6}:)?(?P<host>\S+)
+# Values:  TEXT
+#
+# Sample log entry
+# 03.10.2016 10:50:01   POP3 Server: Authentication failure for httpd, connecting host 58.1.245.164: Password not found in the Name and Address Book entry or password did not verify
+# 03.10.2016 10:51:19   POP3 Server: Authentication failure for noreply@github.com, connecting host 80.122.241.130: Password not found in the Name and Address Book entry or password did not verify
+
+failregex = .* POP3 Server: Authentication failure for .* connecting host <HOST>: Password not found
```

Comment:
The CI-error says:
> AssertionError: Following regexp of "domino-pop3" contains greedy catch-all before `<HOST>`, that is not hard-anchored at end or has not precise sub expression after `<HOST>`:
> `.* POP3 Server: Authentication failure for .* connecting host <HOST>: Password not found`

I've extended our test-cases to cover exactly such situations.
Although your expression can work, but is vulnerable (see abovementioned error):
- multiple catch-all before `<HOST>` token 
- neither it is anchored nor it is precise enough;